modules/ts/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 set(the_description "The ts module")
 
-if(IOS)
+if(IOS OR WINRT)
   ocv_module_disable(ts)
 endif()
```

Comment:
Same as with ```imgcodecs``` this requires extra efort to make it work. Special concern here is that as you know WinRT apps are executed within ```appcontainer``` and are very limited in the ways it can interact with outer conext. This applies e.g. to console applications that do not provide its output in the same simple way you'd expect from regular win32 app. All OCV tests are gtest based and basically are console apps that provide test results in sdtout. That said making it work with WinRT is not a trivial task. 

Again this process is underway, you can take a look at work in progress in [MSOpenCV/test](https://github.com/MSOpenTech/opencv/tree/test) branch, mind this is still WIP. This is as well planned to be a part of the upcoming contributions.